src/io/cscape_result_io.jl

```diff
@@ -9,13 +9,13 @@ struct CScapeResultSet <: ResultSet
     name::String
     RCP::String
 
-    site_ids
-    site_area::Vector{Float64}
-    site_max_coral_cover::Vector{Float64}
-    site_centroids
+    loc_ids
+    loc_area::Vector{Float64}
+    loc_max_coral_cover::Vector{Float64}
```

Comment:
`location_max_coral_cover` in line 128 gets assigned to the field `loc_max_coral_cover` in the results set structure. The name`location_max_coral_cover` is the local variable name and doesn't effect the name used to access it from the result structure. 

```julia
# Load results
rs = ADRIA.load_results(CScapeResultSet, "<path>")

# Get max_coral_cover for each location
data = rs.loc_max_coral_cover

# Error
data = rs.location_max_coral_cover
```

Reply:
It all looks good to me then